src/main/java/org/apache/maven/buildcache/BuildCacheMojosExecutionStrategy.java

```diff
@@ -104,9 +106,9 @@ public void execute( List<MojoExecution> mojoExecutions,
                 mojoExecutionRunner.run( mojoExecution );
             }
             cacheState = cacheConfig.initialize();
-            if ( cacheState == INITIALIZED )
+            if ( cacheState == INITIALIZED || skipCacheLookup )
             {
-                result = cacheController.findCachedBuild( session, project, mojoExecutions );
+                result = cacheController.findCachedBuild( session, project, mojoExecutions, skipCacheLookup );
```

Comment:
if (skipLookup) lookup :-) 
I expected config just to return `DISABLED` state for this module

Reply:
@AlexanderAshitkin renamed to skipCache